flink-table/flink-sql-client/src/test/resources/test-sql-client-catalogs.yaml

```diff
@@ -155,7 +155,6 @@ catalogs:
   - name: hivecatalog
     type: hive
     test: test
-    hive-version: 2.3.4
```

Comment:
I don't think this one should be removed. It's there to verify the version provided by user takes precedence over the version detected in classpath.